addons/dragging/functions/fnc_canCarry.sqf

```diff
@@ -11,35 +11,36 @@
  * Can the unit carry the object? <BOOL>
  *
  * Example:
- * [player, cursorTarget] call ace_dragging_fnc_canCarry
+ * [player, cursorObject] call ace_dragging_fnc_canCarry
```

Comment:
Why `cursorObject` anywhere? That's way too fiddly compared to `cursorTarget` and returns irrelevant stuff like map props.